bundles/binding/org.openhab.binding.plclogo/src/main/java/org/openhab/binding/plclogo/internal/PLCLogoBinding.java

```diff
@@ -212,14 +208,14 @@ protected void execute() {
                         }
 
                         if (config.isSet()) {
```

Comment:
Still no idea why we need it? If we initialize the lastValue with something invalid, for example MAX_INT,
next check, will fail on first call. setLastValue will set "right" value to the lastValue and we are fine.
Miss i something?